code/datums/components/barber.dm

```diff
@@ -758,17 +754,15 @@ ABSTRACT_TYPE(/datum/action/bar/barber)
 		..()
 
 /datum/action/bar/barber/haircut
-	cut = "cut"
-	cuts = "cuts"
-	cutting = "cutting"
-
 	getHairStyles()
 		return get_available_custom_style_types(filter_type=/datum/customization_style/hair)
 
 /datum/action/bar/barber/shave
 	cut = "shave"
 	cuts = "shaves"
 	cutting = "shaving"
+	hair = "face"
+	cut_sound = 'sound/impact_sounds/Flesh_Cut_1.ogg'
```

Comment:
That's definitely a "you fucked up and stabbed them" kind of sound, not a successful shave kind of sound.